onnxruntime/core/providers/cuda/cuda_execution_provider.h

```diff
@@ -200,8 +203,8 @@ class CUDAExecutionProvider : public IExecutionProvider {
     // Cuda graph with multi threads will be supported in the future, so cuda_graph_
     // is put under PerThreadContext.
     CUDAGraph cuda_graph_;
-    bool is_graph_captured_ = false;
     int regular_run_count_before_graph_capture_ = 0;
+    CudaGraphAnnotation_t cuda_graph_annotation_id_ = 0;
```

Comment:
Is this needed ? I thought we have one in the CUDAGraphManager ? Also, doesn't seem to be safe to cache a run level entry in the EP instance.

Reply:
I forgot to remove it

Reply:
Actually, this variable is needed for keeping PerThreadContext's interface unchanged since other eps are using them